### ISSUE TITLE
[Feature request]

### ISSUE THREAD

**Issue by HenkaN94** [Josh5/unmanic#89]
Blacklist failed files after x attempts (obviously a toggle and ability to clear blacklisted files).

Feels completely unnecessary to keep on trying the same file over and over if you have it set to scan the library if the file keeps on failing.

**Comment by Josh5** [comment]
I have added a fix in commit 2f2355163de6afbdcd30078e9e7c4105d9872e1b
This prevents files from being added when they have failed in the history.

Once this is merged into staging I will close this issue.
